### PR TITLE
feat(cloudwatch): Additional Properties for Cloudwatch AlarmStatusWidget

### PR DESCRIPTION
**Pull request by markussiebert**
I've added the missing properties from the Alarm Status Widget

see: https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html#CloudWatch-Dashboard-Properties-Alarm-Widget-Object

fixes: #19386 

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by markussiebert** [comment]
Hi @madeline-k maybe you can review this small change?

File: packages/@aws-cdk/aws-cloudwatch/lib/alarm-status-widget.ts [hunk 1]
```diff
@@ -27,6 +54,24 @@ export interface AlarmStatusWidgetProps {
...
+  readonly sortBy?: AlarmStatusWidgetSortBy;
+
+  /**
+   * Use this field to filter the list of alarms displayed in the widget to only those alarms currently in the specified states.
+   * You can specify one or more alarm states in the value for this field.
+   * The alarm states that you can specify are ALARM, INSUFFICIENT_DATA, and OK.
+   *
+   * If you omit this field or specify an empty array, all the alarms specifed in alarms are displayed.
+   *
+   * @default - [] - all the alarms specifed in alarms are displayed.
```
Comment (madeline-k) [inline packages/@aws-cdk/aws-cloudwatch/lib/alarm-status-widget.ts:72]:
```suggestion
   * @default -  all the alarms specified in alarms are displayed.
```

File: packages/@aws-cdk/aws-cloudwatch/README.md [hunk 2]
```diff
@@ -436,6 +436,20 @@ dashboard.addWidgets(
...
 
+An alarm status widget only showing firing alarms, sorted by state and timestamp:
+
+```ts
+declare const dashboard: cloudwatch.Dashboard;
+declare const errorAlarm: cloudwatch.Alarm;
+
+dashboard.addWidgets(new cloudwatch.AlarmStatusWidget({
+  title: "Errors",
+   alarms: [errorAlarm],
```
Comment (madeline-k) [inline packages/@aws-cdk/aws-cloudwatch/README.md:447]:
```suggestion
  alarms: [errorAlarm],
```

File: packages/@aws-cdk/aws-cloudwatch/lib/alarm-status-widget.ts [hunk 3]
```diff
@@ -27,6 +54,24 @@ export interface AlarmStatusWidgetProps {
    * @default 3
    */
   readonly height?: number;
+
+  /**
+   * Specifies how to sort the alarms in the widget.
+   *
+   * @default DEFAULT
```
Comment (madeline-k) [inline packages/@aws-cdk/aws-cloudwatch/lib/alarm-status-widget.ts:61]:
```suggestion
   * @default - alphabetical order
```

**Review by madeline-k — CHANGES_REQUESTED** [review]
Looks pretty good @markussiebert! I just have a few minor comments.

File: packages/@aws-cdk/aws-cloudwatch/lib/alarm-status-widget.ts [hunk 4]
```diff
@@ -56,6 +101,8 @@ export class AlarmStatusWidget extends ConcreteWidget {
         properties: {
           title: this.props.title ? this.props.title : 'Alarm Status',
           alarms: this.props.alarms.map((alarm) => alarm.alarmArn),
+          states: this.props.states ?? [],
+          sortBy: this.props.sortBy ?? AlarmStatusWidgetSortBy.DEFAULT,
```
[the remark this answers is not in the document]
Comment (markussiebert) [reply packages/@aws-cdk/aws-cloudwatch/lib/alarm-status-widget.ts:105]: ok, will change this

**Review by madeline-k — APPROVED** [review]
Thanks for contributing, @markussiebert! Approved!

**Pull request merged**